proto/bragi/common.proto

```diff
@@ -49,20 +49,28 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
+}
+
 message Match {
   string match_urn = 1;
-  string tournament_urn = 2;
+  string tournament_urn = 2 [deprecated=true];
   Sport sport = 3;
-  string home_team_urn = 4;
-  string away_team_urn = 5;
+  string home_team_urn = 4 [deprecated=true];
+  string away_team_urn = 5 [deprecated=true];
   uint32 home_score = 6;
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  Tournament tournament  = 10;
+  Team home_team = 11;
+  Team away_team = 12;
```

Comment:
I don't like this change. When we implement racing sports, we'll encounter another problem, as these sports don't have a Home and Away team but multiple teams. When making these changes, we should consider future requirements.